pykechain/models/widgets/widgets_manager.py

```diff
@@ -1421,9 +1460,17 @@ def add_signature_widget(
         :raises IllegalArgumentError: when incorrect arguments are provided
         :raises APIError: When the widget could not be created.
         """
-        attachment_property: "AttachmentProperty" = _retrieve_object(
+        attachment_property: "Property" = _retrieve_object(
```

Comment:
Why remove the type check for "AttachmentProperty"?

Reply:
Copy-paste that went wrong